Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -8,14 +8,25 @@ import UIKit
 /// An `UIView` subclass that shows summary and preview information about a given channel.
 public typealias ChatChannelListItemView = _ChatChannelListItemView<NoExtraData>
 
+/// The content of this view.
+///
+/// Initializing `Content` as `_ChatChannelListItemView<NoExtraData>.Content<NoExtraData>`
+/// probably pushes the compiler too far and makes `Content` ambiguous according to `SwiftUIRepresentable.Content`
+public typealias ChatChannelListItemViewContent = _ChatChannelListItemView<NoExtraData>.Content<NoExtraData>
+
```

Comment:
please see my comment about Content not having to be generic

Reply:
Fixed.